text/0001-cdk-update.md

```diff
@@ -105,16 +90,16 @@ Deploying DataStack
 Done!
 ```
 
-In addition to running in a one shot mode, the `cdk update` command also has a
-`--watch` command line option that enable it to monitor the assets on disk and
-perform an update when they change.
+In addition to running in a one shot mode, the `cdk deploy --accelerated`
```

Comment:
Will we also have a short form, like `cdk deploy -a -w` ?

Reply:
That's a good question; the space of single-letter flags is much smaller than the long option ones, so that depends on if we think it's worth claiming one for this feature. I think "no" for `--watch` -- that's not going to be run as often, by its very nature -- and I lean towards "no" for `--accelerate` if we use that, because I can't see an unambiguous short name for it (`-a` would be better as a shortcut for `--all` for example)

Reply:
+1